tools/src/main/java/org/apache/kafka/tools/streams/StreamsGroupCommandOptions.java

```diff
@@ -76,8 +121,14 @@ public StreamsGroupCommandOptions(String[] args) {
             .withRequiredArg()
             .describedAs("streams group")
             .ofType(String.class);
+        topicOpt = parser.accepts("topic", TOPIC_DOC)
```

Comment:
According to the KIP, this should be --input-topics, and allow multiple topics to be passed.